packages/react-composites/src/composites/CallComposite/CallScreen.tsx

```diff
@@ -99,7 +99,7 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
         callErrorHandler('removed');
       }
     }
-  }, [adapter, callErrorHandler, endedCall]);
```

Comment:
The `adapeter` isn't referenced inside this `useEffect` block. Can you help me understand how this was causing the bug?

Reply:
Oh, wait. You _removed_ the adapter from the dependencies. I still don't understand why this was causing the bug though. Why was adapter changing, and if it was, why was it triggering a page change?

Reply:
nvm, this is just clean up. The fix is in CallScreen.

Got it.